features/project/source/browse_files.feature

```diff
@@ -33,6 +33,13 @@ Feature: Project Source Browse Files
     And I click on "Commit Changes"
     Then I am redirected to the new file
     And I should see its new content
+    
+  @javascript
+  Scenario: I can upload file and commit
+    Given I click on "new file" link in repo
+    Then I can see new file page
+    And I can see "upload existing one"
+    And I click on "upload existing one"
```

Comment:
This may be overkill, but here's a good article on how to test Dropzone interaction with Capybara:

http://blog.paulrugelhiatt.com/rails/testing/capybara/dropzonejs/2014/12/29/test-dropzonejs-file-uploads-with-capybara.html